bigbluebutton-html5/imports/ui/components/video-provider/video-provider-graphql/queries.ts

```diff
@@ -83,9 +83,16 @@ export const VIEWERS_IN_WEBCAM_COUNT_SUBSCRIPTION = gql`
   }
 `;
 
-export const VIDEO_STREAMS_USERS_SUBSCRIPTION = gql`
-  subscription VideoStreamsUsers {
-    user {
+export const GRID_USERS_SUBSCRIPTION = gql`
+  subscription GridUsers($exceptUserIds: [String]!, $limit: Int!) {
+    user(
+      where: {
+        userId: {
+          _nin: $exceptUserIds
+        },
+      },
+      limit: $limit
```

Comment:
```suggestion
      limit: $limit,
      order_by: {nameSortable: asc, userId: asc}
```

Once you set a `limit`, probably it's a good idea to include an `order_by` to make sure you will bring the correct users (I guess it will be ordered by name, right?)

Reply:
I think it's not necessary. Grid items don't participate in the sorting/pagination process. So there is no ordering requirement. But it's a nice idea, what's your opinion on bringing them by join time?

Reply:
Even if you don't care about the ordering, it's important to set it!
Otherwise the users returned will be random and it can make the data change all the time... it will increase the traffic flow!